go-controller/pkg/ovn/gress_policy.go

```diff
@@ -128,6 +139,9 @@ func (gp *gressPolicy) addPortPolicy(portJSON *knet.NetworkPolicyPort) {
 }
 
 func (gp *gressPolicy) addIPBlock(ipblockJSON *knet.IPBlock) {
+	gp.ipBlock.cidr = ipblockJSON.CIDR
+	gp.ipBlock.except = append(gp.ipBlock.except, ipblockJSON.Except...)
```

Comment:
`addIPBlock()` can be called multiple times on a given `gressPolicy`, which is why you need an array of `ipBlock`. (A `gressPolicy` represents a single `knet.NetworkPolicyIngressRule`/`knet.NetworkPolicyEgressRule`, which can have multiple `NetworkPolicyPeer`s, each with an `ipBlock`.)

And then below you will need to generate a separate ACL for each `ipBlock` in the policy.

Reply:
done